### PR TITLE
#90757 Prevent Erroneous Deletion of Linked Posts On Event-Preview

### PR DESCRIPTION
**Pull request by ggwicz**
**Ticket:** [**#90757**](http://central.tri.be/issues/90757)

### CODE REVIEW

File: readme.txt [hunk 1]
```diff
@@ -313,8 +313,9 @@ The plugin is made with love by [Modern Tribe Inc](http://m.tri.be/2s).
 * New - Made the "events" and "event" slugs translatable by WPML and other multilingual plugins [95026]
 * New - Introduced the `tribe_events_query_force_local_tz` filter to allow for forcing non-UTC event start and end times in Tribe__Events__Query [92948]
 * Fix - Allow The Events Calendar REST API to be disabled using the `tribe_events_rest_api_enabled` filter [97209]
-* Tweak - Added some more context to the labeling of the "Number of events per page" option (thanks to Todd H. for highlighting this label) [73659]
 * Fix - Remove the errant `div.tribe-events-single-section` on the single event view when there is no venue [97615]
+* Fix - Prevent newly-created venues and organizers from being deleted when saving and previewing drafts of new events (thanks Matt R. for reporting this issue) [90757]
+* Tweak - Added some more context to the labeling of the "Number of events per page" option (thanks to Todd H. for highlighting this label) [73659]
```
Comment (bordoni) [inline readme.txt]: Is this changelog entry relevant to this?

Comment (ggwicz) [reply readme.txt]: I was rearranging things in the readme, these aren't "new" with this PR just rearranged.

**Comment by ggwicz** [comment]
addressed merge conflicts; ready for another review 🔍
